packages/overlay/src/AbstractOverlay.ts

```diff
@@ -18,9 +18,10 @@ import type {
     OverlayOptionsV1,
```

Comment:
Also some comments on this component would be appreciated

Reply:
Added.